src/main/java/com/cisco/trex/stateful/api/lowlevel/ASTFProfileSpecial.java

```diff
@@ -0,0 +1,58 @@
+package com.cisco.trex.stateful.api.lowlevel;
+
+import com.google.gson.JsonObject;
+
+import java.util.List;
+
+/**
+ *
+ * This class provide a separate method to fetch cached json command. When user use it to create
+ * profiles,all the profiles will not interact with each other.So user can create all the profiles
+ * first and then start all of them.
+ * This class have no impact on the class AstfProfile.
+ * For creating one profile and then start it , we can use AstfProfile as before , and can also use
+ * this Class.
+ * For creating multi profile and then start all of them , we should use this class.
+ */
+public class ASTFProfileSpecial extends ASTFProfile {
```

Comment:
considering not extending ASTFProfile, instead creating a ASTFProfile instance internally, like in construtor, then this class can only expose the necessary methods out to user, ex., toJson()